fitnessllm_dataplatform/stream/strava/services/api_interface.py

```
@@ -0,0 +1,118 @@
+from fitnessllm_dataplatform.stream.strava.cloud_utils import get_strava_storage_path
+from fitnessllm_dataplatform.stream.strava.entities.enums import StravaKeys, StravaURLs, StravaStreams
+from fitnessllm_dataplatform.utils.cloud_utils import get_secret, write_json_to_storage
+from fitnessllm_dataplatform.utils.logging_utils import logger
+from stravalib import Client
+
+class RedisConnect:
+    pass
+
+
+class StravaAPIInterface(APIInterface):
+    def __init__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)